pom.xml

```
@@ -29,8 +29,8 @@
         <spring-boot-starter-test>2.6.5</spring-boot-starter-test>
         <maven.javadoc.plugin>3.3.0</maven.javadoc.plugin>
         <maven.gpg.plugin>1.6</maven.gpg.plugin>
-        <aerospike>5.1.11</aerospike>
-        <aerospike-reactor>5.1.11</aerospike-reactor>
+        <aerospike>6.1.2</aerospike>
+        <aerospike-reactor>6.1.2</aerospike-reactor>
         <embedded-aerospike>2.1.4</embedded-aerospike>
```

```suggestion
        <embedded-aerospike>2.2.8</embedded-aerospike>
```
please update embedded aerospike

---

@ijusti We will update all necessary dependencies in a single PR before releasing the next version of Spring Data Aerospike (including embedded-aerospike dependency). 
Thanks for noticing 🙂